Makefile

```diff
@@ -396,3 +397,130 @@ image-mnist-job-test-build: ## Build container image with the MNIST job.
 .PHONY: image-mnist-job-test-push
 image-mnist-job-test-push: image-mnist-job-test-build ## Push container image with the MNIST job.
 	podman push ${MNIST_JOB_TEST_IMG}
+
+# RHOAI/ODH related resources installation
+
+# Basic Usage
+# all-in-one will create all resources necessary to create GPU enabled ML workloads via OpenShift AI
+# Users have the choice between installing RHOAI and ODH
+# For RHOAI use `make all-in-one` and to remove all of the operators run `make delete-all-in-one`
+# For ODH use `make all-in-one -e USE_RHOAI=false` and to remove all of the operators run `make delete-all-in-one -e USE_RHOAI=false`
+
+##@ all-in-one
+.PHONY: all-in-one
+all-in-one:
+	@echo -e "\n ==> Installing Everything needed for distributed AI platform on OpenShift cluster \n"
+	-make install-nfd-operator
+	-make install-ai-platform-operator
+	-make install-nvidia-operator
+
+.PHONY: delete-all-in-one
+delete-all-in-one:
+	@echo -e "\n ==> Removing Everything needed for distributed AI platform on OpenShift cluster \n"
+	-make delete-nfd-operator
+	-make delete-ai-platform-operator
+	-make delete-nvidia-operator
+
+##@ general
+.PHONY: delete-ai-platform-operator
+delete-ai-platform-operator:
+ifeq ($(USE_RHOAI), true) ## Delete RHOAI Operator
+	-make delete-rhoai-operator
+else ## Delete Open Data Hub Operator
+	-make delete-opendatahub-operator
+endif
+
+.PHONY: install-ai-platform-operator
+install-ai-platform-operator:
+ifeq ($(USE_RHOAI), true) ## Delete RHOAI Operator
+	-make install-rhoai-operator
+else ## Delete Open Data Hub Operator
+	-make install-opendatahub-operator
+endif
+
+.PHONY: delete-rhoai-operator
+delete-rhoai-operator: ## Delete RHOAI Operator
+	@echo -e "\n ==> Deleting OpenShift AI Operator \n"
```

Comment:
It would be good to delete also DSC and DSCI CRs

Reply:
possibly to wait until they are deleted before uninstalling operator - that way RHOAI can clean up everything what was created in cluster